api/CSSStyleDeclaration.json

```
@@ -544,6 +544,55 @@
             "deprecated": false
           }
         }
+      },
+      "@@iterator": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/CSSStyleDeclaration/@@iterator",
+          "support": {
+            "chrome": {
+              "version_added": "51",
+              "notes": "Before Chrome 51, we can iterate it after calling <code>Array.from()</code> to transfer it."
```

I don't think that we should include this note. Two reasons:

1. This is very likely true (excepting the version number) for every browser, so we'd probably have to copy and maintain it for each browser.
2. The browsers this applies to are all more than five years old. It's rather obscure at this point.

That said, if you think it's still worth mentioning, I suggest you open an issue (or PR) for the https://developer.mozilla.org/docs/Web/API/CSSStyleDeclaration page on mdn/content. Since it's a strategy that would make sense for any browser, it could just be a general suggestion on the page.

```suggestion
              "version_added": "51"
```